src/modules/launcher/Plugins/Microsoft.Plugin.Registry/plugin.json

```diff
@@ -0,0 +1,12 @@
+{
+  "ID": "303417D927BF4C97BCFFC78A123BE0C8",
+  "ActionKeywords": [ "HK", "hk", "Hk", "hK" ],
```

Comment:
why is the keyword HK versus maybe a symbol of some sort?    HK could be used by someone for a file name such as Hong Kong or HKD for their dollar.

How about **!** or something like that?

Reply:
Why not use `hklm` or `hkcu` or one of the others?

Reply:
actually, ! would be better for services.  

- %% maybe?

here are the current items in use: https://docs.microsoft.com/en-us/windows/powertoys/run#action-key

Action key | Action
-- | --
= | Calculator only. Example =2+2
? | File searching only. Example ?road to find roadmap.txt
. | Installed app searching only. Example .code to get Visual Studio Code
// | URLs only. Example //docs.microsoft.com to have your default browser go to https://docs.microsoft.com
< | Running processes only. Example <outlook to find all processes that contain outlook
> | Shell command only. Example >ping localhost to do a ping query

Reply:
For the service plugin we use _ at the moment.

Reply:
What you think about `\` (Backslash)

The Regedit Editor write in the address line
`Computer\HKEY_ ...`
so, when we reduce this to 
`\HKEY_ ...`
then is the `\` more intuitive

Reply:
I personally don't like `\` as action key for the reg plugin. `\` remind me on fie paths. What about using `_` or `$`? (For `_` please see my comment on #.)

Reply:
We should not use `$`,
IMHO a `$` looks like a environmental variable or a windows special folder (idea for a new plugin 💡)

Reply:
What you think about `:` or `::` so it would look more like PowerShell
`Get-Item -Path Registry::HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion ... `

Reply:
Don't like `:` as action key.

So I think we can use `%` like @crutkas suggested or `_` (if not used for services).
Or what about `&` or `#` (as synonym for a number setting like DWORD)?

Reply:
Actually `:` seems a good choice.

Reply:
I have change it to `:`